verilog/analysis/verilog_linter.cc

```diff
@@ -383,22 +383,48 @@ void GetLintRuleDescriptionsHelpFlag(std::ostream* os,
   }
 }
 
-void GetLintRuleDescriptionsMarkdown(std::ostream* os) {
+void GetLintRuleFile(std::ostream *os) {
+  // Set up the map.
+  auto rule_map = analysis::GetAllRuleDescriptions();
+  for (const auto &rule_id : analysis::kDefaultRuleSet) {
+    rule_map[rule_id].default_enabled = true;
+  }
+
+  // Print all the rules
+  for (const auto &rule : rule_map) {
+    *os << (rule.second.default_enabled ? "+" : "-");
+    *os << rule.second.descriptor.name;
+
+    bool first = true;
+    for (auto it = rule.second.descriptor.param.begin();
+         it != rule.second.descriptor.param.end(); ++it) {
+      *os << (first ? "=" : ";");
+      *os << it->name << ":";
+      *os << it->default_value;
+
+      first = false;
+    }
+
+    *os << std::endl;
```

Comment:
Just make that "\n". `std::endl` unnecessarily flushes the stream.

Reply:
Thanks for pointing the difference out. Done.